README.md

```diff
@@ -491,6 +491,7 @@ Table of Contents
   * [foxpass.com](https://www.foxpass.com/) — Hosted LDAP and RADIUS. Easy per-user logins to servers, VPNs and wireless networks. Free for 10 users
   * [globalsign.com](https://www.globalsign.com/en/ssl/ssl-open-source/) — Free SSL certificates for Open Source
   * [Have I been pwned?](https://haveibeenpwned.com) — REST API for fetching the information on the breaches.
+  * [Hdiv](https://github.com/hdiv/hdiv) - Application security monitoring and protection (RASP) for web applications and APIs.
```

Comment:
Whats the Software As A Service offering here and what in that offering is free?

Reply:
There's nothing free in their SaaS model. 😅

Reply:
It is an Open Source Java RASP solution to add advanced protection capabilities to the applications, integrating with the most commonly used development frameworks, like Spring, JSF, and others.
We think it is interesting to include in the list, but I will remove the PR if you see it is out of scope.

Reply:
The list is for SaaS offerings. So I don’t think it’s a fit.